addons/l10n_in_tds_entry/wizard/tds_entry_wizard.py

```diff
@@ -0,0 +1,101 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import models, fields, api, Command
+
+
+class TdsEntryWizard(models.TransientModel):
+    _name = 'tds.entry.wizard'
+    _description = "Create Tds Entry Wizard"
+
+
+    move_id = fields.Many2one('account.move', required=True)
+    date = fields.Date(string="Entry Date", default=fields.Date.today(), copy=False)
+    tax_id = fields.Many2one('account.tax', string="TDS Section", readonly=False, domain=[('tax_group_id.name', '=', 'TDS')])
+    tds_amount = fields.Monetary(string="TDS Amount", compute='_compute_tds_amount', currency_field='currency_id')
+    journal_id = fields.Many2one(
+        comodel_name='account.journal',
+        string='Journal',
+        required=True,
+        domain=[('type', '=', 'general')],
+        default=lambda self: self.env['account.journal'].search([('type', '=', 'general')], limit=1)
+    )
+    reference = fields.Char(string="Reference", readonly=True)
+    currency_id = fields.Many2one(related="move_id.currency_id", string="Currency")
+    amount_untaxed = fields.Monetary(string="Base Amount", currency_field='currency_id')
+    partner_id = fields.Many2one(related="move_id.partner_id", string="Partner")
+    company_id = fields.Many2one(related="move_id.company_id", string="Company")
+    l10n_in_pan = fields.Char(related="move_id.partner_id.l10n_in_pan", string="PAN", store=True)
+    company_type = fields.Selection(related="move_id.partner_id.company_type")
+
+    @api.model
+    def default_get(self, fields_list):
+        defaults = super().default_get(fields_list)
+
+        move_id = self.env['account.move'].browse(self.env.context['active_id'])
+        defaults['move_id'] = move_id.id
```

Comment:
can we use context on the button to set the default move_id?

Reply:
okay